doc/release_process.rst

```diff
@@ -3,38 +3,34 @@ RSMTool Release Process
 
 This process is only meant for the project administrators, not users and developers.
 
-1. Run ``tests/update_files.py`` to make sure that all test data in the new release have correct experiment ids and filenames. If any (non-model) files need to be changed this should be investigated before the branch is released. 
+1. Run ``python tests/update_files.py`` to make sure that all test data in the new release have correct experiment ids and filenames. If any (non-model) files need to be changed this should be investigated before the branch is released. 
```

Comment:
The full command would also include arguments --tests and --outputs - by giving `python` prefix do we imply that this is the actual command rather than reference to a script?

Reply:
Good catch. I will fix this.